def raise_exception():
    raise TypeError("TypeError!")